tests/UnitTests.vala

```diff
@@ -3,15 +3,247 @@ namespace Vamp {
         public static int main (string[] args) {
             Test.init (ref args);
 
-            Test.add_func ("/vamp/foo", () => {
-                assert_null (null);
+            Test.add_func ("/vamp/deserialize_basic_config", () => {
+                string package_contents;
+                try {
+                    bool did_open = FileUtils.get_contents (TestConfig.TEST_PACKAGE_FILE, out package_contents);
+                    if (!did_open) {
+                        error ("Failed to get contents from: %s".printf (TestConfig.TEST_PACKAGE_FILE));
+                    }
```

Comment:
The docs says that you will only receive `false` in case an error occurs, but if that happens, Vala will catch it with the `try..catch` block, so this `if` will never actually evaluate to a `false`.